mass_mailing_sending_queue/i18n/es.po

```diff
@@ -0,0 +1,156 @@
+msgid "Last Updated on"
+msgstr "Última actualización en"
+
+#. module: mass_mailing_sending_queue
+#: model:ir.model,name:mass_mailing_sending_queue.model_mail_mass_mailing
+msgid "Mass Mailing"
+msgstr "Envío masivo"
+
+#. module: mass_mailing_sending_queue
+#: view:mail.mass_mailing.sending:mass_mailing_sending_queue.view_mail_mass_mailing_sending_search
+#: field:mail.mass_mailing.sending,mass_mailing_id:0
+msgid "Mass mailing"
+msgstr "Envío masivo"
+
+#. module: mass_mailing_sending_queue
+#: model:ir.ui.menu,name:mass_mailing_sending_queue.menu_mass_mailing_sending
+#: field:mail.mail.statistics,mass_mailing_sending_id:0
+#: view:mail.mass_mailing.sending:mass_mailing_sending_queue.view_mail_mass_mailing_sending_form
+#: view:mail.mass_mailing.sending:mass_mailing_sending_queue.view_mail_mass_mailing_sending_search
+msgid "Mass mailing sending"
+msgstr "Envío de correo masivo"
+
+#. module: mass_mailing_sending_queue
+#: model:ir.actions.act_window,name:mass_mailing_sending_queue.action_view_mail_mass_mailing_sending
+#: view:mail.mass_mailing.sending:mass_mailing_sending_queue.view_mail_mass_mailing_sending_tree
+msgid "Mass mailing sendings"
+msgstr "Envíos de correo masivo"
+
+#. module: mass_mailing_sending_queue
+#: field:mail.mass_mailing,pending:0
+msgid "Pending"
+msgstr "Pendiente"
+
+#. module: mass_mailing_sending_queue
+#: code:addons/mass_mailing_sending_queue/models/mail_mass_mailing.py:23
+#: code:addons/mass_mailing_sending_queue/models/mail_mass_mailing_sending.py:67
+#, python-format
+msgid "Please select recipients."
+msgstr "Por favor, seleccione algún destinatario"
+
+#. module: mass_mailing_sending_queue
+#: field:mail.mass_mailing.sending,pending:0
+msgid "Recipients pending"
+msgstr "Destinatarios pendientes"
+
+#. module: mass_mailing_sending_queue
+#: view:mail.mass_mailing.sending:mass_mailing_sending_queue.view_mail_mass_mailing_sending_search
+msgid "Running"
+msgstr "Ejecutándose"
+
+#. module: mass_mailing_sending_queue
+#: view:mail.mass_mailing.sending:mass_mailing_sending_queue.view_mail_mass_mailing_sending_search
+#: selection:mail.mass_mailing.sending,state:0
+msgid "Sending"
+msgstr "Enviando"
+
+#. module: mass_mailing_sending_queue
+#: view:mail.mass_mailing.sending:mass_mailing_sending_queue.view_mail_mass_mailing_sending_search
+#: selection:mail.mass_mailing.sending,state:0
+msgid "Sent"
+msgstr "Enviado"
+
+#. module: mass_mailing_sending_queue
+#: field:mail.mass_mailing.sending,state:0
+msgid "State"
+msgstr "Estado"
+
+#. module: mass_mailing_sending_queue
+#: view:mail.mass_mailing:mass_mailing_sending_queue.view_mail_mass_mailing_form
+msgid "recipients pending for creating email."
+msgstr "destinatarios pendientes de crear el correo electrónico."
+
+#. module: mass_mailing_sending_queue
+#: view:mail.mass_mailing:mass_mailing_sending_queue.view_mail_mass_mailing_form
+msgid "{'invisible': [('scheduled', '=', 0), ('pending', '=', 0)]}"
+msgstr ""
```

Comment:
Don't leave this empty, because then the translation will be considered. Put the same as source of remove the entry.